priv/repo/migrations/20200710132150_create_votes.exs

```diff
@@ -0,0 +1,27 @@
+defmodule LiquidVoting.Repo.Migrations.CreateVotes do
+  use Ecto.Migration
+
+  def change do
+    create table(:votes, primary_key: false) do
+      add :uuid, :uuid, primary_key: true
+      add :yes, :boolean, default: false, null: false
+      add :weight, :integer, default: 1
+
+      add :participant_uuid,
+          references(:participants, column: :uuid, type: :uuid, on_delete: :nothing)
+
+      add :proposal_url, :text, default: false, null: false
+      add :organization_uuid, :uuid
+
+      timestamps()
+    end
+
+    create index(:votes, [:participant_uuid])
+    create index(:votes, [:proposal_url])
```

Comment:
same here

Reply:
participant_uuid + :proposal_url + organization_uuid (and delete current 2 indices)?

Reply:
no, because each index covers different queries - one is all votes for a participant regardless of proposal and the other all votes for a proposal regardless of participant.

so `[:participant_uuid, :organization_uuid]` for the first, and `[:proposal_ulr, :organization_uuid]` for the second one

Reply:
for clarity: all the suggested new compound indices (in the 3 migrations) are not unique indices, just indices, right?

Reply:
right, they're all for finding collections of records, not a single unique one